backend/de.metas.adempiere.adempiere/base/src/main/java-gen/org/compiere/model/I_M_Product.java

```diff
@@ -1824,6 +1823,29 @@ public interface I_M_Product
 	ModelColumn<I_M_Product, org.compiere.model.I_S_Resource> COLUMN_S_Resource_ID = new ModelColumn<>(I_M_Product.class, "S_Resource_ID", org.compiere.model.I_S_Resource.class);
 	String COLUMNNAME_S_Resource_ID = "S_Resource_ID";
 
+	/**
+	 * Set FLO ID.
+	 * ID granted by FLOCERT for companies to put it on their products.
+	 *
+	 * <br>Type: String
+	 * <br>Mandatory: false
+	 * <br>Virtual Column: false
+	 */
+	void setth173_FLO_ID (@Nullable java.lang.String th173_FLO_ID);
```

Comment:
as talked => column should be named FLO_Identifier or sth similar (not `th173_`, not `_ID`)